olad/OladHTTPServer.cpp

```diff
@@ -508,6 +513,44 @@ int OladHTTPServer::ModifyUniverse(const HTTPRequest *request,
   return MHD_YES;
 }
 
+/**
+ * @brief Set plugin state.
+ * @param request the HTTPRequest
+ * @param response the HTTPResponse
+ * @returns MHD_NO or MHD_YES
+ */
+int OladHTTPServer::SetPluginState(const HTTPRequest *request,
+                                   HTTPResponse *response) {
+  if (request->CheckParameterExists(HELP_PARAMETER)) {
+    return ServeUsage(response,
+                      "POST state=[enable|disable], "
+                      // "plugin_ids=[a comma separated list of plugin ids]");
+                      "plugin_ids=[a plugin id]");
+  }
+
+  string state_string = request->GetPostParameter("state");
+
+  bool state;
+  if (!StringToBoolTolerant(state_string, &state)) {
+    OLA_INFO << "Invalid state " << state_string;
+    return ServeHelpRedirect(response);
+  }
+
+  string plugin_id_string = request->GetPostParameter("plugin_ids");
```

Comment:
We can fan out here if we really need to support changing multiple plugins at once, but I'd avoid it if we can.